plone/namedfile/scaling.py

```diff
@@ -17,7 +17,12 @@
 from zope.traversing.interfaces import ITraversable, TraversalError
 from zope.publisher.interfaces import IPublishTraverse, NotFound
 from zope.app.file.file import FileChunk
-from plone.protect.interfaces import IDisableCSRFProtection
+
+try:
+    # Soft dependency to make this package work without plone.protect
+    from plone.protect.interfaces import IDisableCSRFProtection
+except ImportError:
```

Comment:
Thanks @mauritsvanrees 
See, not discussed to the end. There is still room for discussion, IMO. However, I've updated the pull-request.